sonar/resources/projects/schema.py

```diff
@@ -133,12 +93,46 @@ def guess_user(self, data, **kwargs):
         return data
 
 
-class ProjectSchemaV1(StrictKeysMixin):
-    """Project schema."""
+class ProjectSchema(BaseRecordSchema):
+    """Schema for records v1 in JSON."""
+
+    metadata = fields.Nested(ProjectMetadataSchema)
+
+    def dump(self, obj, *args, **kwargs):
+        """Dump object.
+
+        Override the parent method to add the documents linked to projects.
+        It was not possible to use the `pre_dump` decorator, because
+        `add_permission` need this property and we cannot be sure that this
```

Comment:
Seems weird.

Reply:
Yes it is. Maybe I can dig to find another solution but it's not related to this PR.